lib/mayaUsd/utils/editRouter.cpp

```diff
@@ -97,7 +95,7 @@ void cacheMayaReference(const PXR_NS::VtDictionary& context, PXR_NS::VtDictionar
 
     auto createCachePrimFn = [stage, dstLayer, dstPrimPath](
                                  const PXR_NS::SdfPath& primPath, bool asReference, bool append) {
-        auto cachePrim = stage->DefinePrim(primPath);
+        auto cachePrim = stage->DefinePrim(primPath, PXR_NS::TfToken("Xform"));
```

Comment:
The cache prim is immediately created with Xform type, rather than trying to set its type later.